.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "dev/helm-charts/pulsar-helm-chart"]
```

Comment:
Why do we need a submodule for this?  

Can't we just add a pulsar repo and install that?  This seems way heavy handed.  We don't really need versioning for the helm charts I don't think.  Not yet at least.

Could you follow this guide to install?

https://pulsar.apache.org/docs/2.11.x/getting-started-helm/

Reply:
Did you read through step one of that guide? It explicitly has the user checkout this repo and run a script from it. Perhaps it's not necessary for our needs, but I'm familiar enough with pulsar yet to determine that.